discounts/javascript/discounts-allocator/default/src/run.graphql.liquid

```diff
@@ -0,0 +1,7 @@
+query RunInput {
+  shop {
+    metafield(namespace: "$app:{{handle | replace: " ", "-" | downcase}}", key: "allocator-function") {
```

Comment:
for other functions we've been using the key `function-configuration`, which I think makes the intent clearer

Reply:
Done!